MexTK/include/devtext.h

```diff
@@ -23,6 +23,7 @@ struct DevText
     char x25;                 // 0x25
     char show_text : 1;       // 0x26
     char show_background : 1; // 0x26
+    char x26_20 : 1;          // 0x26
```

Comment:
I realize now there are multiple conventions used in m-ex for naming bitfields. Some use a 2 digit hex mask (80, 40, etc) and some use a bit number 1-8. Sometimes the mask is useful for fields that have bits of multiple lengths. It may be worth changing some to be more consistent in the future.